src/commands/support/apmodes.ts

```diff
@@ -0,0 +1,19 @@
+import { MessageCommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+const MANAGED_SELECTED_IMAGE = 'https://cdn.discordapp.com/attachments/1234165731319353436/1258331400709472256/New_Project_1.png?ex=6687a7f0&is=66865670&hm=462f8a417207e18117703c42295da293497dc2895f88f30d068c07f216ea5259&';
```

Comment:
Could you please add the image to the `assets` folder under the support category? The image URL will then be accessible through `${imageBaseUrl}/support/<file_name.png>`

You can find an example of this here: https://github.com/flybywiresim/discord-bot/blob/staging/src/commands/support/dlss.ts